src/python/gudhi/alpha_complex.pyx

```diff
@@ -53,10 +53,12 @@ cdef class AlphaComplex:
 
     """
 
-    cdef Alpha_complex_interface * thisptr
+    cdef Alpha_complex_interface * this_ptr
+    cdef bool fast
```

Comment:
Isn't that redundant with `this_ptr->fast_version_`?

Reply:
Yes, no need to store this information. Modified on 58cb456a